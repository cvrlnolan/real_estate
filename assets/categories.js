export const categoryOptions = [
    { key: 'Mansion', text: 'Mansion', value: 'Mansion' },
    { key: 'Family House', text: 'Family House', value: 'Family House' },
    { key: 'Apartment', text: 'Apartment', value: 'Apartment' },
    { key: 'Loft', text: 'Loft', value: 'Loft' },
    { key: 'Studio', text: 'Studio', value: 'Studio' },
    { key: 'Roomlet', text: 'Roomlet', value: 'Roomlet' },
]