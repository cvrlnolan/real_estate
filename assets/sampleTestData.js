const estates = [
    //Data copied directly from MongoDB Atlas personal Cluster & Database
    {
        "_id": "611e520ffa5eab373dd9ba37",
        "title": "Modern Apartment Downtown Los Angeles",
        "price": "2500",
        "status": "rent",
        "address": "2465 Evergreen Lane",
        "province": "California",
        "postal_code": "90042",
        "country": "United States",
        "category": "Apartment",
        "bedrooms": "2",
        "baths": "1",
        "surface_area": "200",
        "property_briefing": "Lorem Ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
        "additional_info": "",
        "cooling": true,
        "heating": true,
        "internet": true,
        "furniture": false,
        "parking": true,
        "email": "madara@konoha.com",
        "telephone": "+237696740298",
        "createdDate": "2021-08-19T12:43:42.180Z",
        "imgUrl": "https://firebasestorage.googleapis.com/v0/b/real-estate-2ff9c.appspot.com/o/Estates?alt=media&token=03b46643-93ed-4057-adf1-b8308cdaabbb",
        "rating": "0",
        "reviews": "3",
        "totalRating": "13"
    },
    {
        "_id": "611e5248fa5eab373dd9ba38",
        "title": "Modern Apartment Downtown Los Angeles",
        "price": "2500",
        "status": "rent",
        "address": "2465 Evergreen Lane",
        "province": "California",
        "postal_code": "90042",
        "country": "United States",
        "category": "Apartment",
        "bedrooms": "2",
        "baths": "1",
        "surface_area": "200",
        "property_briefing": "Lorem Ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
        "additional_info": "",
        "cooling": true,
        "heating": true,
        "internet": true,
        "furniture": false,
        "parking": true,
        "email": "madara@konoha.com",
        "telephone": "+237696740298",
        "createdDate": "2021-08-19T12:44:42.041Z",
        "imgUrl": "https://firebasestorage.googleapis.com/v0/b/real-estate-2ff9c.appspot.com/o/Estates?alt=media&token=a3fbec61-d9e2-484c-be45-f1f667d1f58f",
        "rating": "0",
        "reviews": "9",
        "totalRating": "40"
    }
]

export { estates }