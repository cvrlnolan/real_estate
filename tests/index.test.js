/**
 * @jest-environment jsdom
 */

import React from "react"
import "@testing-library/jest-dom"
import { render, screen } from "@testing-library/react"
import renderer from "react-test-renderer"
import Home from "../pages/index"
import { estates } from "@/assets/sampleTestData"

describe("Index Page", () => {
    it("should render correctly", async () => {
        render(<Home estatesData={estates} />)
        screen.debug()
    })

    it("should match snapshot", () => {
        const tree = renderer.create(<Home estatesData={estates} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})

