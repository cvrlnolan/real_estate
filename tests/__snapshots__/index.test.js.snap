// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page should match snapshot 1`] = `
Array [
  <div
    className="css-1m65r4j"
  >
    <div
      className="css-jfo7z"
    >
      <button
        aria-label="Open Menu"
        className="chakra-button css-1mh58fi"
        onClick={[Function]}
        type="button"
      >
        <svg
          aria-hidden={true}
          className="chakra-icon css-onkibi"
          focusable={false}
          viewBox="0 0 24 24"
        >
          <path
            d="M 3 5 A 1.0001 1.0001 0 1 0 3 7 L 21 7 A 1.0001 1.0001 0 1 0 21 5 L 3 5 z M 3 11 A 1.0001 1.0001 0 1 0 3 13 L 21 13 A 1.0001 1.0001 0 1 0 21 11 L 3 11 z M 3 17 A 1.0001 1.0001 0 1 0 3 19 L 21 19 A 1.0001 1.0001 0 1 0 21 17 L 3 17 z"
            fill="currentColor"
          />
        </svg>
      </button>
      <div
        className="chakra-stack css-16awuyr"
      >
        <div
          className="css-0"
        >
          RealEstate
        </div>
        <nav
          className="chakra-stack css-t14v28"
        >
          <a
            className="css-3xxwyh"
            href="/"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Listings
          </a>
          <a
            className="css-3xxwyh"
            href="https://github.com/cvrlnolan/real_estate"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Repository
          </a>
        </nav>
      </div>
      <div
        className="css-70qvj9"
      >
        <button
          className="chakra-button css-1br927b"
          href="/estate/add"
          onClick={[Function]}
          onMouseEnter={[Function]}
          type="button"
        >
          <span
            className="chakra-button__icon css-1wh2kri"
          >
            <svg
              aria-hidden={true}
              className="chakra-icon css-onkibi"
              focusable={false}
              viewBox="0 0 24 24"
            >
              <path
                d="M0,12a1.5,1.5,0,0,0,1.5,1.5h8.75a.25.25,0,0,1,.25.25V22.5a1.5,1.5,0,0,0,3,0V13.75a.25.25,0,0,1,.25-.25H22.5a1.5,1.5,0,0,0,0-3H13.75a.25.25,0,0,1-.25-.25V1.5a1.5,1.5,0,0,0-3,0v8.75a.25.25,0,0,1-.25.25H1.5A1.5,1.5,0,0,0,0,12Z"
                fill="currentColor"
              />
            </svg>
          </span>
          Add Estate
        </button>
        <button
          aria-label="theme_button"
          className="chakra-button css-10a8y2i"
          type="button"
        >
          <svg
            aria-hidden={true}
            className="chakra-icon css-onkibi"
            focusable={false}
            viewBox="0 0 24 24"
          >
            <g
              fill="none"
              stroke="currentColor"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="2"
            >
              <circle
                cx="12"
                cy="12"
                r="5"
              />
              <path
                d="M12 1v2"
              />
              <path
                d="M12 21v2"
              />
              <path
                d="M4.22 4.22l1.42 1.42"
              />
              <path
                d="M18.36 18.36l1.42 1.42"
              />
              <path
                d="M1 12h2"
              />
              <path
                d="M21 12h2"
              />
              <path
                d="M4.22 19.78l1.42-1.42"
              />
              <path
                d="M18.36 5.64l1.42-1.42"
              />
            </g>
          </svg>
        </button>
      </div>
    </div>
  </div>,
  <div
    className="css-2rlxtj"
  >
    <div
      className="chakra-container css-4cl8wm"
    >
      <div
        className="css-1fuoiyw"
      >
        <a
          className="css-p6s8gx"
          href="/estate/611e520ffa5eab373dd9ba37"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <div
            className="css-16hc176"
          >
            <div
              className="css-1yzea99"
            >
              <div
                style={
                  Object {
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "left": 0,
                    "margin": 0,
                    "overflow": "hidden",
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                  }
                }
              >
                <img
                  alt="card_image"
                  data-nimg="fill"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style={
                    Object {
                      "border": "none",
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "display": "block",
                      "height": 0,
                      "left": 0,
                      "margin": "auto",
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                      "minHeight": "100%",
                      "minWidth": "100%",
                      "objectFit": "cover",
                      "objectPosition": undefined,
                      "padding": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                      "width": 0,
                    }
                  }
                />
                <noscript>
                  <img
                    alt="card_image"
                    data-nimg="fill"
                    decoding="async"
                    loading="lazy"
                    sizes="100vw"
                    src="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3D03b46643-93ed-4057-adf1-b8308cdaabbb&w=3840&q=75"
                    srcSet="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3D03b46643-93ed-4057-adf1-b8308cdaabbb&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3D03b46643-93ed-4057-adf1-b8308cdaabbb&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3D03b46643-93ed-4057-adf1-b8308cdaabbb&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3D03b46643-93ed-4057-adf1-b8308cdaabbb&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3D03b46643-93ed-4057-adf1-b8308cdaabbb&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3D03b46643-93ed-4057-adf1-b8308cdaabbb&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3D03b46643-93ed-4057-adf1-b8308cdaabbb&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3D03b46643-93ed-4057-adf1-b8308cdaabbb&w=3840&q=75 3840w"
                    style={
                      Object {
                        "border": "none",
                        "bottom": 0,
                        "boxSizing": "border-box",
                        "display": "block",
                        "height": 0,
                        "left": 0,
                        "margin": "auto",
                        "maxHeight": "100%",
                        "maxWidth": "100%",
                        "minHeight": "100%",
                        "minWidth": "100%",
                        "objectFit": "cover",
                        "objectPosition": undefined,
                        "padding": 0,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                        "width": 0,
                      }
                    }
                  />
                </noscript>
              </div>
            </div>
            <div
              className="chakra-stack css-n21gh5"
            >
              <div
                className="css-gg4vpm"
              >
                <p
                  className="chakra-text css-13ub55k"
                >
                  Apartment
                </p>
                <p
                  className="chakra-text css-11ox3m3"
                >
                  2
                   Bedrooms • 
                  1
                   Baths • 
                  200
                   sqft
                </p>
              </div>
              <h2
                className="chakra-heading css-1r8d57x"
              >
                Modern Apartment Downtown Los Angeles
              </h2>
              <p
                className="chakra-text css-1duy3yu"
              >
                $
                2500
                /mo.
              </p>
              <div
                className="css-k008qs"
              >
                <svg
                  className="chakra-icon css-1t3c8dd"
                  focusable={false}
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M23.555,8.729a1.505,1.505,0,0,0-1.406-.98H16.062a.5.5,0,0,1-.472-.334L13.405,1.222a1.5,1.5,0,0,0-2.81,0l-.005.016L8.41,7.415a.5.5,0,0,1-.471.334H1.85A1.5,1.5,0,0,0,.887,10.4l5.184,4.3a.5.5,0,0,1,.155.543L4.048,21.774a1.5,1.5,0,0,0,2.31,1.684l5.346-3.92a.5.5,0,0,1,.591,0l5.344,3.919a1.5,1.5,0,0,0,2.312-1.683l-2.178-6.535a.5.5,0,0,1,.155-.543l5.194-4.306A1.5,1.5,0,0,0,23.555,8.729Z"
                    fill="currentColor"
                  />
                </svg>
                <svg
                  className="chakra-icon css-1t3c8dd"
                  focusable={false}
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M23.555,8.729a1.505,1.505,0,0,0-1.406-.98H16.062a.5.5,0,0,1-.472-.334L13.405,1.222a1.5,1.5,0,0,0-2.81,0l-.005.016L8.41,7.415a.5.5,0,0,1-.471.334H1.85A1.5,1.5,0,0,0,.887,10.4l5.184,4.3a.5.5,0,0,1,.155.543L4.048,21.774a1.5,1.5,0,0,0,2.31,1.684l5.346-3.92a.5.5,0,0,1,.591,0l5.344,3.919a1.5,1.5,0,0,0,2.312-1.683l-2.178-6.535a.5.5,0,0,1,.155-.543l5.194-4.306A1.5,1.5,0,0,0,23.555,8.729Z"
                    fill="currentColor"
                  />
                </svg>
                <svg
                  className="chakra-icon css-1t3c8dd"
                  focusable={false}
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M23.555,8.729a1.505,1.505,0,0,0-1.406-.98H16.062a.5.5,0,0,1-.472-.334L13.405,1.222a1.5,1.5,0,0,0-2.81,0l-.005.016L8.41,7.415a.5.5,0,0,1-.471.334H1.85A1.5,1.5,0,0,0,.887,10.4l5.184,4.3a.5.5,0,0,1,.155.543L4.048,21.774a1.5,1.5,0,0,0,2.31,1.684l5.346-3.92a.5.5,0,0,1,.591,0l5.344,3.919a1.5,1.5,0,0,0,2.312-1.683l-2.178-6.535a.5.5,0,0,1,.155-.543l5.194-4.306A1.5,1.5,0,0,0,23.555,8.729Z"
                    fill="currentColor"
                  />
                </svg>
                <svg
                  className="chakra-icon css-1t3c8dd"
                  focusable={false}
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M23.555,8.729a1.505,1.505,0,0,0-1.406-.98H16.062a.5.5,0,0,1-.472-.334L13.405,1.222a1.5,1.5,0,0,0-2.81,0l-.005.016L8.41,7.415a.5.5,0,0,1-.471.334H1.85A1.5,1.5,0,0,0,.887,10.4l5.184,4.3a.5.5,0,0,1,.155.543L4.048,21.774a1.5,1.5,0,0,0,2.31,1.684l5.346-3.92a.5.5,0,0,1,.591,0l5.344,3.919a1.5,1.5,0,0,0,2.312-1.683l-2.178-6.535a.5.5,0,0,1,.155-.543l5.194-4.306A1.5,1.5,0,0,0,23.555,8.729Z"
                    fill="currentColor"
                  />
                </svg>
                <svg
                  className="chakra-icon css-1t3c8dd"
                  focusable={false}
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M23.555,8.729a1.505,1.505,0,0,0-1.406-.98H16.062a.5.5,0,0,1-.472-.334L13.405,1.222a1.5,1.5,0,0,0-2.81,0l-.005.016L8.41,7.415a.5.5,0,0,1-.471.334H1.85A1.5,1.5,0,0,0,.887,10.4l5.184,4.3a.5.5,0,0,1,.155.543L4.048,21.774a1.5,1.5,0,0,0,2.31,1.684l5.346-3.92a.5.5,0,0,1,.591,0l5.344,3.919a1.5,1.5,0,0,0,2.312-1.683l-2.178-6.535a.5.5,0,0,1,.155-.543l5.194-4.306A1.5,1.5,0,0,0,23.555,8.729Z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              <p
                className="chakra-text css-ax9bdp"
              >
                2465 Evergreen Lane
                , 
                90042
                 
                California
              </p>
              <p
                className="chakra-text css-nu9dau"
              >
                21
                 day(s) ago
              </p>
            </div>
          </div>
        </a>
        <a
          className="css-p6s8gx"
          href="/estate/611e5248fa5eab373dd9ba38"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <div
            className="css-16hc176"
          >
            <div
              className="css-1yzea99"
            >
              <div
                style={
                  Object {
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "left": 0,
                    "margin": 0,
                    "overflow": "hidden",
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                  }
                }
              >
                <img
                  alt="card_image"
                  data-nimg="fill"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style={
                    Object {
                      "border": "none",
                      "bottom": 0,
                      "boxSizing": "border-box",
                      "display": "block",
                      "height": 0,
                      "left": 0,
                      "margin": "auto",
                      "maxHeight": "100%",
                      "maxWidth": "100%",
                      "minHeight": "100%",
                      "minWidth": "100%",
                      "objectFit": "cover",
                      "objectPosition": undefined,
                      "padding": 0,
                      "position": "absolute",
                      "right": 0,
                      "top": 0,
                      "width": 0,
                    }
                  }
                />
                <noscript>
                  <img
                    alt="card_image"
                    data-nimg="fill"
                    decoding="async"
                    loading="lazy"
                    sizes="100vw"
                    src="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3Da3fbec61-d9e2-484c-be45-f1f667d1f58f&w=3840&q=75"
                    srcSet="/_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3Da3fbec61-d9e2-484c-be45-f1f667d1f58f&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3Da3fbec61-d9e2-484c-be45-f1f667d1f58f&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3Da3fbec61-d9e2-484c-be45-f1f667d1f58f&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3Da3fbec61-d9e2-484c-be45-f1f667d1f58f&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3Da3fbec61-d9e2-484c-be45-f1f667d1f58f&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3Da3fbec61-d9e2-484c-be45-f1f667d1f58f&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3Da3fbec61-d9e2-484c-be45-f1f667d1f58f&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Ffirebasestorage.googleapis.com%2Fv0%2Fb%2Freal-estate-2ff9c.appspot.com%2Fo%2FEstates%3Falt%3Dmedia%26token%3Da3fbec61-d9e2-484c-be45-f1f667d1f58f&w=3840&q=75 3840w"
                    style={
                      Object {
                        "border": "none",
                        "bottom": 0,
                        "boxSizing": "border-box",
                        "display": "block",
                        "height": 0,
                        "left": 0,
                        "margin": "auto",
                        "maxHeight": "100%",
                        "maxWidth": "100%",
                        "minHeight": "100%",
                        "minWidth": "100%",
                        "objectFit": "cover",
                        "objectPosition": undefined,
                        "padding": 0,
                        "position": "absolute",
                        "right": 0,
                        "top": 0,
                        "width": 0,
                      }
                    }
                  />
                </noscript>
              </div>
            </div>
            <div
              className="chakra-stack css-n21gh5"
            >
              <div
                className="css-gg4vpm"
              >
                <p
                  className="chakra-text css-13ub55k"
                >
                  Apartment
                </p>
                <p
                  className="chakra-text css-11ox3m3"
                >
                  2
                   Bedrooms • 
                  1
                   Baths • 
                  200
                   sqft
                </p>
              </div>
              <h2
                className="chakra-heading css-1r8d57x"
              >
                Modern Apartment Downtown Los Angeles
              </h2>
              <p
                className="chakra-text css-1duy3yu"
              >
                $
                2500
                /mo.
              </p>
              <div
                className="css-k008qs"
              >
                <svg
                  className="chakra-icon css-1t3c8dd"
                  focusable={false}
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M23.555,8.729a1.505,1.505,0,0,0-1.406-.98H16.062a.5.5,0,0,1-.472-.334L13.405,1.222a1.5,1.5,0,0,0-2.81,0l-.005.016L8.41,7.415a.5.5,0,0,1-.471.334H1.85A1.5,1.5,0,0,0,.887,10.4l5.184,4.3a.5.5,0,0,1,.155.543L4.048,21.774a1.5,1.5,0,0,0,2.31,1.684l5.346-3.92a.5.5,0,0,1,.591,0l5.344,3.919a1.5,1.5,0,0,0,2.312-1.683l-2.178-6.535a.5.5,0,0,1,.155-.543l5.194-4.306A1.5,1.5,0,0,0,23.555,8.729Z"
                    fill="currentColor"
                  />
                </svg>
                <svg
                  className="chakra-icon css-1t3c8dd"
                  focusable={false}
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M23.555,8.729a1.505,1.505,0,0,0-1.406-.98H16.062a.5.5,0,0,1-.472-.334L13.405,1.222a1.5,1.5,0,0,0-2.81,0l-.005.016L8.41,7.415a.5.5,0,0,1-.471.334H1.85A1.5,1.5,0,0,0,.887,10.4l5.184,4.3a.5.5,0,0,1,.155.543L4.048,21.774a1.5,1.5,0,0,0,2.31,1.684l5.346-3.92a.5.5,0,0,1,.591,0l5.344,3.919a1.5,1.5,0,0,0,2.312-1.683l-2.178-6.535a.5.5,0,0,1,.155-.543l5.194-4.306A1.5,1.5,0,0,0,23.555,8.729Z"
                    fill="currentColor"
                  />
                </svg>
                <svg
                  className="chakra-icon css-1t3c8dd"
                  focusable={false}
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M23.555,8.729a1.505,1.505,0,0,0-1.406-.98H16.062a.5.5,0,0,1-.472-.334L13.405,1.222a1.5,1.5,0,0,0-2.81,0l-.005.016L8.41,7.415a.5.5,0,0,1-.471.334H1.85A1.5,1.5,0,0,0,.887,10.4l5.184,4.3a.5.5,0,0,1,.155.543L4.048,21.774a1.5,1.5,0,0,0,2.31,1.684l5.346-3.92a.5.5,0,0,1,.591,0l5.344,3.919a1.5,1.5,0,0,0,2.312-1.683l-2.178-6.535a.5.5,0,0,1,.155-.543l5.194-4.306A1.5,1.5,0,0,0,23.555,8.729Z"
                    fill="currentColor"
                  />
                </svg>
                <svg
                  className="chakra-icon css-1t3c8dd"
                  focusable={false}
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M23.555,8.729a1.505,1.505,0,0,0-1.406-.98H16.062a.5.5,0,0,1-.472-.334L13.405,1.222a1.5,1.5,0,0,0-2.81,0l-.005.016L8.41,7.415a.5.5,0,0,1-.471.334H1.85A1.5,1.5,0,0,0,.887,10.4l5.184,4.3a.5.5,0,0,1,.155.543L4.048,21.774a1.5,1.5,0,0,0,2.31,1.684l5.346-3.92a.5.5,0,0,1,.591,0l5.344,3.919a1.5,1.5,0,0,0,2.312-1.683l-2.178-6.535a.5.5,0,0,1,.155-.543l5.194-4.306A1.5,1.5,0,0,0,23.555,8.729Z"
                    fill="currentColor"
                  />
                </svg>
                <svg
                  className="chakra-icon css-1t3c8dd"
                  focusable={false}
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M23.555,8.729a1.505,1.505,0,0,0-1.406-.98H16.062a.5.5,0,0,1-.472-.334L13.405,1.222a1.5,1.5,0,0,0-2.81,0l-.005.016L8.41,7.415a.5.5,0,0,1-.471.334H1.85A1.5,1.5,0,0,0,.887,10.4l5.184,4.3a.5.5,0,0,1,.155.543L4.048,21.774a1.5,1.5,0,0,0,2.31,1.684l5.346-3.92a.5.5,0,0,1,.591,0l5.344,3.919a1.5,1.5,0,0,0,2.312-1.683l-2.178-6.535a.5.5,0,0,1,.155-.543l5.194-4.306A1.5,1.5,0,0,0,23.555,8.729Z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              <p
                className="chakra-text css-ax9bdp"
              >
                2465 Evergreen Lane
                , 
                90042
                 
                California
              </p>
              <p
                className="chakra-text css-nu9dau"
              >
                21
                 day(s) ago
              </p>
            </div>
          </div>
        </a>
      </div>
    </div>
  </div>,
]
`;
